### PR TITLE
completed objects exercise

### PR DESCRIPTION
**Pull request by Eric-Fried**
prep javascript objects

### CODE REVIEW

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# prep-javascript-objects-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What data types can object properties hold?
+
+- Describe the syntax (structure) of object-literals in JavaScript.
+
```
Comment (shawnkost) [inline prep-javascript-objects/prep-javascript-objects-quiz-notes.md]: Don't forget to answer the quiz questions!

**Review by shawnkost — CHANGES_REQUESTED** [review]
Code looks good Eric! Just need to answer the quiz questions

File: prep-javascript-objects/prep-javascript-objects-quiz-notes.md (hunk 1 above)
Comment (Eric-Fried) [reply prep-javascript-objects/prep-javascript-objects-quiz-notes.md]: I just re committed with the quiz answered

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**